components/02-elements/form/radio/Radio.stories.js

```diff
@@ -20,7 +21,9 @@ storiesOf('Elements/Form/Radio', module)
           :legend="options.legend" 
           :options="options.options" 
           :name="options.name"
+          v-model="selected"
         />
+        Selected option: {{ selected }}
```

Comment:
I don't think we need it, we can preview this value in Storybook, via https://github.com/Sambego/storybook-state or https://github.com/storybooks/storybook/tree/master/addons/knobs

Reply:
But can be done as a enhancement later, so let's just create a task for this and do to next issue :)